bootstrap.sh

```diff
@@ -54,11 +67,19 @@ install_packages()
 
 compile_proto()
 {
-    python3 -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. ./silk/tools/pb/visualize_grpc.proto
+    $python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. ./silk/tools/pb/visualize_grpc.proto
+}
+
+setup_venv()
+{
+    python3 -m venv env
```

Comment:
we should use $python

Reply:
I think it won't work here yet since the venv is created by this command. That venv path doesn't exist yet.